Source/ASDisplayNode+Yoga.mm

```diff
@@ -36,11 +36,15 @@ @interface ASDisplayNode (YogaInternal)
 - (ASSizeRange)_locked_constrainedSizeForLayoutPass;
 @end
 
+@interface ASLayout (YogaInternal)
+@property (nonatomic, getter=isFlattened) BOOL flattened;
```

Comment:
@nguyenhuy could you explain why the diff to this file in this other PR is correct?
 https://github.com/TextureGroup/Texture/pull/395/files#diff-599cfec3d0b870d11b2965c36e2b6a72 

I haven't been able to study the change in detail, but I wanted to be certain that flattening will still be performed correctly for Yoga nodes with this removed.